dotnet/VERSIONS.md

```diff
@@ -36,24 +36,6 @@ This is the scheme: `OsMajor.OsMinor.InternalRelease[-prereleaseX]+sha.1b2c3d4`.
         * Example: `iOS 15.1.123-rt` (and nothing else). This makes these
           versions exactly 3 characters longer than the release version, which
           is hopefully enough to avoid MAX_PATH issues on Windows.
-* Build metadata: Required Hash
```

Comment:
Instead of removing this chunk of text, we could make it optional:

```
* Build metadata: Optional Hash
[...]
```

Reply:
Updated